api/views.py

```diff
@@ -166,6 +168,47 @@ def get_queryset(self):
             result__branch_name=branch,
         )
 
+
+@api_view(["GET"])
+def dynamic_benchmark_summary(request):
+    branch = request.query_params.get('branch')
+    environment = request.query_params.get('environment')
+    benchmarks = request.query_params.getlist('benchmarks')
+
+    testjobs = benchmarks_models.TestJob.objects.filter(environment__identifier=environment)
+    testjobs = testjobs.select_related('result')
+    testjobs = testjobs.filter(result__branch_name=branch)
+    if settings.IGNORE_GERRIT is False:
+        testjobs = testjobs.filter(result__gerrit_change_number=None)
```

Comment:
I think the meaning of IGNORE_GERRIT is to allow any values on gerrit related fields, not only None.